utils/fetch_external_bangs.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+import sys
+import json
+import re
+from os.path import realpath, dirname, join
+
+import requests
+
+# set path
+sys.path.append(realpath(dirname(realpath(__file__)) + '/../'))
+
+from searx import searx_dir
+
+
+# from https://duckduckgo.com/newbang
+URL_BV1 = 'https://duckduckgo.com/bv1.js'
+
+RE_BANG_VERSION = re.compile('\/bang\.v([0-9]+)\.js')
```

Comment:
The string of the regexp contains anomalous backslashes .. it is better to use raw strings for regexp (`r'\/bang\.v([0-9]+)\.js'`). BTW it easy to find issues like that with pylint.  Here is my patch to do a pylint:

```diff
diff --git a/Makefile b/Makefile
index 13f965b6..fbf5f94b 100644
--- a/Makefile
+++ b/Makefile
@@ -194,7 +194,8 @@ PYLINT_FILES=\
        searx/engines/google_news.py \
        searx/engines/google_videos.py \
        searx/engines/google_images.py \
-       searx/engines/mediathekviewweb.py
+       searx/engines/mediathekviewweb.py \
+       utils/fetch_external_bangs.py
 
 test.pylint: pyenvinstall
        $(call cmd,pylint,$(PYLINT_FILES))
diff --git a/utils/fetch_external_bangs.py b/utils/fetch_external_bangs.py
old mode 100644
new mode 100755
index 99f820a9..72155cba
--- a/utils/fetch_external_bangs.py
+++ b/utils/fetch_external_bangs.py
@@ -1,5 +1,7 @@
 #!/usr/bin/env python
 
+# pylint: disable=C0116
+

```

With `make test.pylint` I got this messages::

```pylint
utils/fetch_external_bangs.py:21:0: W1401: Anomalous backslash in string: '\/'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
utils/fetch_external_bangs.py:21:6: W1401: Anomalous backslash in string: '\.'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
utils/fetch_external_bangs.py:21:17: W1401: Anomalous backslash in string: '\.'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
utils/fetch_external_bangs.py:27:0: C0303: Trailing whitespace (trailing-whitespace)
utils/fetch_external_bangs.py:41:15: C0303: Trailing whitespace (trailing-whitespace)
utils/fetch_external_bangs.py:1:0: C0114: Missing module docstring (missing-module-docstring)
utils/fetch_external_bangs.py:15:0: C0413: Import "from searx import searx_dir" should be placed at the top of the module (wrong-import-position)
utils/fetch_external_bangs.py:89:16: W0621: Redefining name 'bangs' from outer scope (line 135) (redefined-outer-name)
```

Reply:
Done